README.md

```diff
@@ -1,6 +1,4 @@
-<<<<<<< HEAD
-# DocBook-Back-End
-=======
+
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍